src/main/features/claim/routes/claim-receipt.ts

```diff
@@ -1,24 +1,21 @@
 import * as express from 'express'
-import * as http from 'http'
```

Comment:
undo this rename please, i.e. put it back to `receipt.ts` to be consistent with response
REC